openhands-workspace/openhands/workspace/docker/workspace.py

```diff
@@ -11,12 +11,6 @@
 
 from pydantic import Field, PrivateAttr, model_validator
 
-from openhands.agent_server.docker.build import (
-    BuildOptions,
-    PlatformType,
-    TargetType,
-    build,
-)
 from openhands.sdk.logger import get_logger
```

Comment:
We should still be able to import TargetType. This probably means we should move these build related logic somewhere up to SDK, what do you think? @OpenHands can you help me do some research and see if this is a good idea, if so, implement this for me